### PR TITLE
Flvstream rtmpdump conflict

### PR DESCRIPTION
**Pull request by Moisan**
Following [issue 42766](https://github.com/Homebrew/homebrew/issues/42766), add conflicts_with between flvstreamer and rtmpdump over rtmpsrv, rtmpsuck and rtmpsuck binaries.

### CODE REVIEW

File: Library/Formula/rtmpdump.rb [hunk 1]
```diff
@@ -18,6 +18,8 @@ class Rtmpdump < Formula
     sha256 "90f87f1c3e8c68385576812bdfadc39152d3bd9166cafb982761d1a6cc915710" => :mountain_lion
   end
 
+  conflicts_with "flvstreamer", :because => "both install 'rtmpsrv', 'rtmpsuck' and 'rtmpsuck' binary"
```
Comment (bfontaine) [inline Library/Formula/rtmpdump.rb]: Is it me or did you mention `'rtmpsuck'` twice?

Comment (DomT4) [inline Library/Formula/rtmpdump.rb]: It's not just you :wink:

Comment (Moisan) [inline Library/Formula/rtmpdump.rb]: Yeah I just copy/pasted the listing on the issue page, I'll fix this.

Comment (bfontaine) [inline Library/Formula/rtmpdump.rb]: Oh yeah it’s duplicated there too.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `flvstreamer: add rtmpdump conflict.`.

**Pull request closed without merging**